lib/resty/shdict/simple.lua

```diff
@@ -21,6 +21,7 @@ function _M.gen_shdict_methods (opts)
     local neg_ttl = opts.negative_ttl
     local disable_shdict = opts.disable_shdict
     local dict_name = opts.dict_name
+    local force_retries = opts.force_retries or 1
```

Comment:
"force_retries" is a bad name. Maybe you mean "max_retries" or "max_tries"? Depending on whether you count the final (successful) try.